README.md

```diff
@@ -18,6 +18,14 @@ Seb's notes : I also changed the pulse parameters for Samsung, update the Panaso
 5. Restart your Arduino ide
 6. Check out the examples
 
+###### Using Git to install library ( Linux )
+`cd ~/Arduino/libraries`
+`git clone https://github.com/markszabo/IRremoteESP8266.git
+`
+**_ To Update to the latest version of the library  _**
```

Comment:
Please take a look at this line in the *preview* tab on github, it isn't formatting well.
i.e.. I think you need to remove the white-space after/before the '_'s.